### PR TITLE
Start/stop xrootd and cmsd together for the origin (SOFTWARE-3544)

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: configs/stash-origin/overrides/xrootd/10-stash-origin-auth-overrides.conf [hunk 1]
```diff
@@ -7,5 +6,6 @@
 #   is down.
 # - Depends on fetch-crl and the reporter script, but doesn't need these
 #   to finish or succeed for startup.
-Wants=fetch-crl-boot.service fetch-crl-cron.service stash-origin-authfile.service stash-origin-authfile.timer
+# - Turn on cmsd; not a hard dependency but you need it to join a federation
```
Comment (bbockelm) [inline configs/stash-origin/overrides/xrootd/10-stash-origin-auth-overrides.conf]:
```suggestion
# - Turn on cmsd; needed to join a federation, but you can shut it off without causing xrootd to turn off also
```

File: configs/stash-origin/overrides/xrootd/10-stash-origin-overrides.conf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[Unit]
```
Comment (bbockelm) [inline configs/stash-origin/overrides/xrootd/10-stash-origin-overrides.conf]: We should either say "see comments in stash-origin-overrides.conf" or duplicate those comments here.

**Comment by matyasselmeci** [comment]
By the way, I had to turn off the authfile generation dependency for testing because the fermicloud VM wasn't registered in topology and the script was erroring out. That's going to be a pain for automated testing...

File: configs/stash-origin/overrides/xrootd/10-stash-origin-overrides.conf (hunk 2 above)
Comment (matyasselmeci) [reply configs/stash-origin/overrides/xrootd/10-stash-origin-overrides.conf]: Fixed.

**Comment by matyasselmeci** [comment]
This is ready for re-review btw

**Pull request merged**